spec/models/game_spec.rb

```
@@ -28,5 +28,52 @@
+        game.generate_prediction!
+      end.to change(Prediction, :count).by(1)
+      prediction = Prediction.last
+      expect(prediction.game).to eq(game)
+      expect(prediction.home_team_snapshot).to eq(home_snapshot)
+      expect(prediction.away_team_snapshot).to eq(away_snapshot)
+    end
+  end
+
+  describe '#finalize' do
+    it 'delegates to ProphetRatings::GameFinalizer' do
+      expect_any_instance_of(ProphetRatings::GameFinalizer).to receive(:call).and_call_original
```

RuboCop (1.75.5)

[convention] Avoid stubbing using `expect_any_instance_of`.

(RSpec/AnyInstance)